lab-taylor/__test__/gallery.test.js

```diff
@@ -0,0 +1,365 @@
+'use strict';
+
+const request = require('superagent');
+const serverToggle = require('../lib/server-toggle.js');
+const server = require('../server.js');
+
+const User = require('../model/user.js');
+const Gallery = require('../model/gallery.js');
+
+require('jest');
+
+const url = 'http://localhost:3000';
+
+const exampleUser = {
```

Comment:
Strongly encourage building a mock module that has methods so you can create a user, remove a user, create a gallery attached to a user, remove the gallery, add a pic to the gallery that is attached to the user, remove the pic...

This will significantly cut down on the amount of repetition in your code. I also would encourage you to look into using Faker to randomize your inputs to give you greater confidence in the ability of your app to handle a wide variety of inputs